src/tasks/tasks.py

```diff
@@ -367,6 +367,56 @@ def get_metrics(self, reset=False):
                 'accuracy': self.scorer2.get_metric(reset)}
 
 
+@register_task('cola-in-domain', rel_path='CoLA/')
+class CoLAInDomainTask(CoLATask):
+    def __init__(self, path, max_seq_len, name="train_in_domain", **kw):
+        super(
+            CoLAInDomainTask,
+            self).__init__(
+            path,
+            max_seq_len,
+            name=name,
+		**kw)
+
+    def load_data(self, path, max_seq_len):
```

Comment:
Fine as-is, but it might be possible to save some boilerplate by adding parameters to `@register_task` that are passed to the constructor.

See https://github.com/jsalt18-sentence-repl/jiant/blob/master/src/tasks/edge_probing.py#L245 and https://github.com/jsalt18-sentence-repl/jiant/blob/master/src/tasks/edge_probing.py#L57 for an example